apps/demo/config/initial-data.ts

```diff
@@ -8,7 +8,7 @@ export const initialData: Record<string, UserData> = {
         props: {
           title: "This page was built with Puck",
           description:
-            "Puck is the self-hosted visual editor for React. Bring your own components and make site changes instantly, without a deploy.",
+            "<p><strong>Puck is the self-hosted visual editor for React.</strong> Bring your own components and make site changes instantly, without a deploy.</p>",
```

Comment:
I actually think we should drop the `strong` here for now

```suggestion
            "<p><strong>Puck is the self-hosted visual editor for React.</strong> Bring your own components and make site changes instantly, without a deploy.</p>",
```